File: app/src/main/java/acr/browser/lightning/search/engine/AskSearch.java
package acr.browser.lightning.search.engine;

import acr.browser.lightning.R;
import acr.browser.lightning.constant.Constants;

/**
 * The Ask search engine.
 */
public class AskSearch extends BaseSearchEngine {

    public AskSearch() {
        super("file:///android_asset/ask.png", Constants.ASK_SEARCH, R.string.search_engine_ask);
    }

}

File: app/src/main/java/acr/browser/lightning/search/engine/AskSearch.kt
package acr.browser.lightning.search.engine

import acr.browser.lightning.R

/**
 * The Ask search engine.
 */
class AskSearch : BaseSearchEngine(
        "file:///android_asset/ask.png",
        "http://www.ask.com/web?qsrc=0&o=0&l=dir&qo=LightningBrowser&q=",
        R.string.search_engine_ask
)
